matrix/common/lambda_handler.py

```diff
@@ -26,8 +32,15 @@ def invoke(self, fn_name: LambdaName, payload: dict):
         :param fn_name: The LambdaName of the function to be invoked
         :param payload: Data passed to invoked lambda
         """
-        self._client.invoke(
-            FunctionName=fn_name.value,
-            InvocationType="Event",
-            Payload=json.dumps(payload).encode(),
-        )
+        try:
+            self._client.invoke(
+                FunctionName=fn_name.value,
+                InvocationType="Event",
+                Payload=json.dumps(payload).encode(),
+            )
+        except ClientError:
+            size = len(json.dumps(payload).encore())
```

Comment:
`encode`?

Reply:
lol clearly didnt test this. thanks good catch